learning_wavelets/tests/models/exact_recon_unet_test.py

```diff
@@ -11,7 +11,7 @@ def test_init():
     K.clear_session()
 
 def test_fit():
-    model=ExactReconUnet(n_output_channels=1, kernel_size=3, layers_n_channels=[4, 8])
+    model=ExactReconUnet(n_output_channels=1, kernel_size=3, layers_n_channels=[4, 8], bn=True, exact_recon=True)
```

Comment:
use `pytest` features to test all cases (see [here](https://docs.pytest.org/en/reorganize-docs/parametrize.html))

Reply:
you didn't solve this actually !

Reply:
Now it's been uploaded